db/migrate/20150205034412_create_urls.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUrls < ActiveRecord::Migration
+  def change
+    create_table :urls do |t|
+      t.string :original_url, null: false, default: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.